src/cli/templates/create/_._npmignore

```diff
@@ -1,2 +1,3 @@
 src/
-webpack.js
\ No newline at end of file
+webpack.config.js
+*.log
```

Comment:
Can you add trailing newline please? According to your own editorconfig :)

Reply:
Done. I think --> 07eadac

Thanks for catching that :)